routes/public.js

```diff
@@ -59,6 +59,47 @@ function PublicRouter(app) {
         });
     })
 
+    router.get('/auth/google', function(req, res, next) {
+        passport.authenticate('google', { scope: ['https://www.googleapis.com/auth/plus.login'] }, function(err, user, info) {
```

Comment:
Are we sure Google wants their own URL for the scope? That goes against the OAuth specs